openeo/rest/job.py

```diff
@@ -424,6 +418,43 @@ def load_bytes(self) -> bytes:
     # TODO: more `load` methods e.g.: load GTiff asset directly as numpy array
 
 
+    def _download_to_file(self, url: str, target: Path, chunk_size: int):
+        head = self.job.connection.head(url, stream=True)
+        if head.ok and 'Accept-Ranges' in head.headers and 'bytes' in head.headers['Accept-Ranges']:
```

Comment:
you can simplify this to
```suggestion
        if head.ok and head.headers.get("Accept-Ranges") == "bytes":
```